standalone_SINGE.sh

```diff
@@ -4,7 +4,7 @@ gene_list=$2
 outdir=$3
 hypefile=$4
 while read arg||[ -n "$arg" ]; do 
-    ./run_SINGE_GLG_Test.sh v94 $data -outdir $outdir $arg
+    ./run_SINGE_GLG_Test.sh /usr/local/MATLAB/MATLAB_Runtime/v94 $data -outdir $outdir $arg
```

Comment:
Is this runtime path specific to the docker environment?

Reply:
Yes, that happens to be where the runtime was installed inside the Docker image.  I'm proposing that we make this another command line argument because the runtime location will be user-specific.